core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,1223 @@
-<div class="oppia-about-page-container">
-  <div class="oppia-static-header">
-    <h1 [innerHTML]="'I18N_ABOUT_PAGE_HEADING' | translate"></h1>
+<div class="about-page">
+  <meta charset="UTF-8">
```

Comment:
Why do we need the meta tag here?

/cc @vojtechjelinek

Reply:
removed